### DISCUSSION TITLE
How do I set the path for the current terminal session (Windows 10)?

### DISCUSSION THREAD

**Discussion by fabianod** [Eugeny/tabby#4778] [Q&A]
I can't find a way to automatically set the path variable for the current terminal session.

I tried to add PATH or SET PATH in the environment option inside "Config file", and also in a custom profile.

I don't want to change the  path for the entire O.S., just for that specific terminal session.

Thanks!

**Comment by Eugeny** [comment]
You can override %PATH% in a profile like this (it's the same for Windows): 
![image](https://user-images.githubusercontent.com/161476/137989201-600d56dc-3ed6-4038-a0c7-b1fdd39b657e.png)

However you'd have to also include all existing PATH entries since it's getting completely overwritten by the new value.

I've filed a separate issue to add variable expansion https://github.com/Eugeny/tabby/issues/4794

**Reply by fabianod** [reply]
> 
> 
> You can override %PATH% in a profile like this (it's the same for Windows)
> 
> However you'd have to also include all existing PATH entries since it's getting completely overwritten by the new value.
> 
> I've filed a separate issue to add variable expansion #4794

Thank you for the answer, however this method does not work for me (I'm on a Windows machine)!

I tried to use PATH and SET PATH, writing the path with a single backslash and also double backslash. And in all the cases when I run the terminal, I end up with the PATH previously set by the system.

![image](https://user-images.githubusercontent.com/10389046/138308210-d06da8fa-9839-457d-8391-f4bef988505c.png)

**Reply by Eugeny** [reply]
The following seems to work on my system - note how the variable name needs to be `Path` and not `PATH`. If I'm using `Path`, both `Path` and `PATH` end up in `set path` output - I'll need to investigate further
![image](https://user-images.githubusercontent.com/161476/138340553-1103971a-60d2-4ebd-8d06-7be76e5f59d7.png)
![image](https://user-images.githubusercontent.com/161476/138340612-ded3b895-186d-4922-924c-ac469ad1f547.png)

**Reply by fabianod** [reply]
yeap, works flawlessly with Path instead of PATH, or path.

That probably happens cause, although the CLI command to set the path isn't case sensitive, looks like the environment variable is!

Windows + R -> sysdm.cpl -> Advanced -> Environment Variables

Again, thanks a lot for your help!
